Cargo.toml

```diff
@@ -99,6 +99,7 @@ serial_test = "3.0"
 similar-asserts = "1.5"
 tempfile = "3.8"
 
-# TODO: Remove once alloy-contract is stable. This is only used in tests for `sol!`.
-[patch.crates-io]
-alloy-sol-macro = { git = "https://github.com/alloy-rs/core", rev = "18b0509950c90d9ec38f25913b692ae4cdd6f227" }
+# TODO: Keep around until alloy-contract is stable.
+# This should only be used in `alloy-contract` tests.
+# [patch.crates-io]
+# alloy-sol-macro = { git = "https://github.com/alloy-rs/core", rev = "18b0509950c90d9ec38f25913b692ae4cdd6f227" }
```

Comment:
Should this be commented?

Reply:
Yes, currently there are no changes but it should be re enabled for testing with network changes